boards/arm/nucleo_l552ze_q/nucleo_l552ze_q_non_secure.dts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2020 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include "stm32l552ze.dtsi"
+#include "arduino_r3_connector.dtsi"
```

Comment:
Connector file include is pure hw description also so could be put in common.dtsi as well.